src/apis/ProxyController.php

```diff
@@ -24,18 +27,33 @@
  * 4. Generate Build.
  * 5. Send build identifier to the client.
  *
+ * @property Module $module
+ *
  * @author Basil Suter <basil@nadar.io>
  * @since 1.0.0
  */
 class ProxyController extends Controller
 {
+    /**
+     * @var Connection
+     * @since 2.0.0
+     */
+    protected $db;
+
     /**
      * @var array A list of tables which will be ignored and can not be synced with the proxy command.
      */
     protected $ignoreTables = [
         'migration', 'admin_proxy_build', 'admin_proxy_machine',
     ];
-    
+
+    public function beforeAction($action)
```

Comment:
Instead of beforeAction, we could just override init() as we do always.

init() {
parent::init();
// ensure db object
}